src/Sleep.js

```diff
@@ -1,26 +1,45 @@
 class Sleep {
-  constructor() {}
-  userSleepData(sleepData, id) {
-    this.userSleep = sleepData.filter(dailySleep => dailySleep.userID === id);
-    return this.userSleep;
+  constructor(sleepSet) {
+    this.sleepSet = sleepSet
```

Comment:
Our constructor now contains one parameter `sleepSet` which will represent our sleep dataset. This will be stored on instantiation, and will be the catalyst for our data flow.

Reply:
ok I was not sure if we needed to store the database in the Sleep class